src/instructlab/model/download.py

```diff
@@ -134,6 +208,57 @@ class OCIDownloader(Downloader):
     We are leveraging OCI v1.1 for this functionality
     """
 
+    def get_metadata(self) -> None:
+        self.metadata.version = self.release
+
+        # Check if skopeo is installed and the version is at least 1.9
+        check_skopeo_version()
```

Comment:
This shouldn't hold up PR, but wondering - is there some way we are indicating skopeo as a requirement for ilab?

Reply:
its not really a requirement because it is only needed for OCI downloads, but folks don't need it if they want to download models from Hugging Face